src/features/pastelConf/pastelConf.ts

```diff
@@ -0,0 +1,43 @@
+import fs from 'fs'
+
+import masternodes from './masternodes'
+
+export const updateDefaultPastelConfig = async (
+  pastelConfPath: string,
+): Promise<number> => {
+  if (!pastelConfPath) {
+    return 2 // pastelConfPath empty
+  }
```

Comment:
Instead of doing this, I think we can try/catch `await fs.promises.readFile(pastelConfPath)`. I don't think someone will willingly call this function with an empty string

Reply:
I have  updated for this one